src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/HighHeatSound.kt

```diff
@@ -0,0 +1,21 @@
+package at.hannibal2.skyhanni.features.mining.crystalhollows
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.MiningAPI
+import at.hannibal2.skyhanni.events.PlaySoundEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object HighHeatSound {
+    private val config get() = SkyHanniMod.feature.mining
+
+    @SubscribeEvent
+    fun onSound(event: PlaySoundEvent) {
+        if (isEnabled()) return
+
+        if (event.soundName == "mob.wolf.panting" && event.pitch == 0.0f && event.volume == 1.0f) event.cancel()
+    }
+
+    private fun isEnabled() = config.muteHighHeat && MiningAPI.inCrystalHollows() && MiningAPI.heat >= 90
```

Comment:
The heat check should be part of `onSound()`, not `IsEnabled()` (low heat still means the feature is enabled)